tests/common/reboot.py

```diff
@@ -353,6 +353,11 @@ def check_reboot_cause(dut, reboot_cause_expected):
     @param dut: The AnsibleHost object of DUT.
     @param reboot_cause_expected: The expected reboot cause.
     """
+    # For kvm testbed, command `show reboot-cause` will return Unknown.
+    # So, return in advance if this check is running on kvm testbed.
+    if dut.facts["asic_type"] == "vs":
+        return True
```

Comment:
How about override `reboot_cause_expected` to `unknown` here?

Reply:
Sure, we can modify this script in this way.